app/models/provider.rb

```diff
@@ -24,4 +32,10 @@ def has_unpublished_changes?
   def is_published?
     content_status == 'published'
   end
+
+private
+
+  def post_base_url
+    "#{Provider.site}#{Provider.path}/%<provider_code>s" % path_attributes
```

Comment:
```suggestion
    "#{Provider.site}#{Provider.path}/%<provider_code>" % path_attributes
```

with out the `s`?

Reply:
It's weird but it's actually required lol